BCI_analysis/pipeline/pipeline_imaging.py

```diff
@@ -63,6 +71,11 @@ def find_conditioned_neuron_idx(session_bpod_file,session_ops_file,fov_stats_fil
         else:
             conditioned_neuron_name  =''
             roi_idx = None
+        if roi_idx != None and metadata['metadata']['hIntegrationRoiManager']['enable'] == metadata['metadata']['hIntegrationRoiManager']['outputChannelsEnabled'] == 'true':
```

Comment:
Sure,
['hIntegrationRoiManager']['enable'] means that in the scanimage GUI the user enabled integration, which means that the intensities of each ROI is calculated for each frame online
metadata['metadata']['hIntegrationRoiManager']['outputChannelsEnabled'] means that the user also enabled to run a function on the extracted ROI fluorescences and output an analog signal to the motor that moves the lickport. 
If any of those are not checked, there is no closed loop control
)

Reply:
@m2kulkarni I have another question:
read_sessionwise_npy(file_path) reads just a single digested session, but in matlab files, we had multiple sessions in the same file, right?
I assume you planned to add another script that finds all the sessions for a FOV and loads them all?

Reply:
I don't have a script that does that since I figured it would be upto the user to choose what sessions they'd want to load. 

Would you want me to write such a script? It would just be a for loop and using `read_sessionwise_npy(file_path)` there.

Reply:
Havent tested this code, might give some errors :p

Reply:
You are right, it's very simple. :D I'll add this!